crates/nu-cli/src/util.rs

```diff
@@ -9,18 +9,35 @@ use nu_protocol::{
 };
 #[cfg(windows)]
 use nu_utils::enable_vt_processing;
-use std::path::PathBuf;
+use std::path::{Path, PathBuf};
 
 // This will collect environment variables from std::env and adds them to a stack.
 //
 // In order to ensure the values have spans, it first creates a dummy file, writes the collected
 // env vars into it (in a "NAME"="value" format, quite similar to the output of the Unix 'env'
 // tool), then uses the file to get the spans. The file stays in memory, no filesystem IO is done.
-pub fn gather_parent_env_vars(engine_state: &mut EngineState) {
-    gather_env_vars(std::env::vars(), engine_state);
+//
```

Comment:
Could you copy-paste this comment to the PR description to explain the motivation behind the PR.

Reply:
Ok, done for that